elements/lisk-p2p/src/constants.ts

```diff
@@ -24,6 +24,7 @@ export const DEFAULT_WS_MAX_MESSAGE_RATE = 100;
 export const DEFAULT_WS_MAX_MESSAGE_RATE_PENALTY = 10;
 export const DEFAULT_RATE_CALCULATION_INTERVAL = 1000;
 export const DEFAULT_WS_MAX_PAYLOAD = 3048576; // Size in bytes
+export const DEFAULT_NONCE_LENGTH_BYTES = 32;
```

Comment:
I think it's my bad, but this actually should be `8` (it was opposite)
32 bytes will be 64 characters, and we want 16 characters